lib/timers.js

```diff
@@ -179,13 +179,13 @@ function decRefCount() {
 
 // Schedule or re-schedule a timer.
 // The item must have been enroll()'d first.
-const active = exports.active = function(item) {
+const active = exports.active = function active(item) {
```

Comment:
should I remove the named functions for these exports?

Reply:
@transitive-bullshit Yes, I think it’s best to keep them as they are if they are assigned as a named property. :)